From khali@linux-fr.org Sun Aug 13 14:34:08 2006
Date: Sun, 13 Aug 2006 23:34:05 +0200
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Subject: i2c-sibyte: Kip Walker is gone
Message-Id: <20060813233405.6471c451.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-sibyte-drop-kip-walker-address.patch

i2c-sibyte: Kip Walker is gone

Kip Walker no longer works at Broadcom, and his e-mail address there
bounces back, so let's drop it.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/i2c/busses/i2c-sibyte.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/i2c/busses/i2c-sibyte.c
+++ gregkh-2.6/drivers/i2c/busses/i2c-sibyte.c
@@ -217,6 +217,6 @@ static void __exit i2c_sibyte_exit(void)
 module_init(i2c_sibyte_init);
 module_exit(i2c_sibyte_exit);
 
-MODULE_AUTHOR("Kip Walker <kwalker@broadcom.com>, Steven J. Hill <sjhill@realitydiluted.com>");
+MODULE_AUTHOR("Kip Walker (Broadcom Corp.), Steven J. Hill <sjhill@realitydiluted.com>");
 MODULE_DESCRIPTION("SMBus adapter routines for SiByte boards");
 MODULE_LICENSE("GPL");
